libexec/swiftenv-versions

```diff
@@ -60,8 +60,23 @@ check_toolchains() {
     for path in "$TOOLCHAIN_DIR"/*; do
       if [[ "$path" == *".xctoolchain" ]] && [ -d "$path" ]; then
         version="$(basename ${path##*/} .xctoolchain)"
+
+        if [ "$version" == "XcodeDefault" ]; then
```

Comment:
Perhaps we can find a better way to find the actual Swift versions here.

#37 #41